modules/programs/pet.nix

```
@@ -0,0 +1,146 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.pet;
+
+  snippetType = types.submodule {
+    options = {
+      description = mkOption {
+        type = types.str;
+        default = "";
+        example = "Show expiration date of SSL certificate";
+        description = ''
+          description of the snippet
```

The description should be a full sentence (including final period). So in this case could have
```suggestion
          Description of the snippet.
```